storybook/test/index.js

```diff
@@ -25,24 +25,6 @@ afterAll( () => {
 
 initStoryshots( {
 	configPath: path.resolve( __dirname, '../' ),
-	test: snapshotWithOptions( ( story ) => ( {
-		// We need to mock refs for some stories which use them.
-		// @see https://reactjs.org/blog/2016/11/16/react-v15.4.0.html#mocking-refs-for-snapshot-testing
-		// @see https://github.com/storybookjs/storybook/tree/master/addons/storyshots/storyshots-core#using-createnodemock-to-mock-refs
-		createNodeMock: ( element ) => {
-			const currentElement =
-				element.type && document.createElement( element.type );
-
-			if ( story.kind === 'Components/ClipboardButton' ) {
-				currentElement.appendChild(
-					document.createElement( 'button' )
-				);
-			}
-			if ( story.kind === 'Components/Popover' ) {
-				const parentElement = document.createElement( 'div' );
-				parentElement.appendChild( currentElement );
-			}
-			return currentElement;
-		},
-	} ) ),
+	integrityOptions: { cwd: __dirname }, // it will start searching from the current directory
```

Comment:
This mocking logic is essential to make tests pass when components use `createRef`.

Reply:
Yes, I was waiting to see which direction #21485 went before spending more time getting all the tests running correctly. I didn't want to spend the time if the decision is to drop storyshots altogether.

Reply:
we can always make those snapshots more useful but adding additional assertions like running a11y checks on the generated content.